bin/configs/kotlin-jvm-spring-webclient.yaml

```diff
@@ -0,0 +1,9 @@
+generatorName: kotlin
+outputDir: samples/client/petstore/kotlin-jvm-spring-webclient
```

Comment:
please add the new folder to .github/workflows/samples-kotlin-client.yaml to sthat CI will run the tests for this new folder.